src/test/java/org/openmrs/contrib/qaframework/automation/LocationsSteps.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.ConfigureMetadataPage;
+import org.openmrs.reference.page.ManageLocationAttributeTypesPage;
+import org.openmrs.reference.page.ManageLocationTagsPage;
+import org.openmrs.reference.page.ManageLocationsPage;
+
+import static org.junit.Assert.assertTrue;
+
+public class LocationsSteps extends Steps {
+
+	private static final String PREFERRED_HANDLER = "Location Field Gen Datatype Handler";
+	private static final String DESCRIPTION = "The location is for expectant mothers";
```

Comment:
What is the description all about

Reply:
Have rather refactored it into the exact method that uses it.